src/Filter.js

```diff
@@ -114,19 +114,20 @@ class Filter extends Component {
             </button>
           </form>
           <div className="filtered-countries">
-            {this.state.filteredCountries.map(country => {
+            {this.state.filteredCountries.sort((a,b) => {
+              return a.population - b.population;
+            }).map((country,index) => {
               return (
```

Comment:
great implementation of this method cole! shows the countries in descending order, rather than alphabetical order as it was before.